apps/jupyter/workload/test.yml

```diff
@@ -0,0 +1,98 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - name: Generate the litmus result CR to reflect SOT (Start of Test) 
```

Comment:
@vibhor995 Can you include common utils for app_deploy, deploy_check, litmus_results in this litmus book? where ever applicable?

Reply:
on it @dargasudarshan